### PR TITLE
feat: POCP-399: add Portugal locale

### PR DESCRIPTION
**Pull request by petro-marik-cko**
## Description

We need to add Portugal locale

## Solution

Add the needed locale

## Notes

Here is google sheers with validated translation: https://docs.google.com/spreadsheets/d/13I2VBpctwj3kE2hfBg2jq3ve5m-vG5tMExDJxARgc_E/edit#gid=0

## Jira Issue

https://checkout.atlassian.net/browse/POCP-399

### CODE REVIEW

**Comment by mateusz-walesiak-cko** [comment]
@duarte-santos-cko tagging you, as I quite can't validate these 😅

**Comment by petro-marik-cko** [comment]
> @duarte-santos-cko tagging you, as I quite can't validate these 😅
Got it, I will ask him.
He is a FR this week and might be super busy

**Comment by petro-marik-cko** [comment]
> > @duarte-santos-cko tagging you, as I quite can't validate these 😅
> 
> Where does these translations come from? IIRC I've contributed in a document

I added this doc with translations in the description.
here is [link](https://docs.google.com/spreadsheets/d/13I2VBpctwj3kE2hfBg2jq3ve5m-vG5tMExDJxARgc_E/edit#gid=0l)

File: src/processout/translator.ts [hunk 1]
```diff
@@ -155,6 +155,55 @@ module ProcessOut {
...
+            "default": "Ocorreu um erro: o pagamento foi recusado.",
+            "card.declined": "O cartão foi recusado.",
+            "card.expired": "O cartão fornecido expirou.",
+            "card.duplicate": "O pagamento não foi concluído. Por favor, tente novamente.",
+            "card.network-failed": "O pagamento não foi concluído. Por favor, tente novamente.",
+            "card.invalid": "O cartão fornecido é inválido.",
+            "card.invalid-name": "O nome do titular do cartão é inválido.",
+            "card.invalid-number": "O número do cartão é inválido.",
+            "card.invalid-date, card.invalid-expiry-date": "A data de validade do cartão é inválida.",
+            "card.invalid-month, card.invalid-expiry-month": "O mês de validade do cartão é inválido.",
```
Comment (roshan-gorasia-cko) [inline src/processout/translator.ts:169]: `"card.invalid-month, card.invalid-expiry-month"` 

Is this valid (I don't know), will it treat those as individual or one whole string?

File: src/processout/translator.ts [hunk 2]
```diff
@@ -155,6 +155,55 @@ module ProcessOut {
...
+            "card.invalid-year, card.invalid-expiry-year": "O ano de validade do cartão é inválido.",
```
Comment (roshan-gorasia-cko) [inline src/processout/translator.ts:170]: Same as above

File: src/processout/translator.ts (hunk 1 above)
Comment (petro-marik-cko) [reply src/processout/translator.ts:169]: No, just checked the final result
It is one key
I will separate

**Review by roshan-gorasia-cko — APPROVED** [review]

**Pull request merged**